crates/scroll/payload/src/builder.rs

```diff
@@ -0,0 +1,70 @@
+use reth_payload_builder::{KeepPayloadJobAlive, PayloadJob, PayloadJobGenerator};
```

Comment:
Same as with the `NoopTransactionPool`, this time because the [TestPayloadJob](https://github.com/scroll-tech/reth/blob/83b2fb9b413b7386d77414e573a8874fb9d83151/crates/payload/builder/src/test_utils.rs#L82) uses Ethereum specific payload types for the GAT's for `PayloadJob`.